frontend/src/download.js

```diff
@@ -20,9 +61,31 @@ $(document).ready(function() {
   });
   $('#download-btn').click(download);
   function download() {
+    storage.totalDownloads += 1;
+    
     const fileReceiver = new FileReceiver();
 
+    const unexpiredFiles = storage.numFiles;
+    let totalUploads = 0;
+    if (storage.has('totalUploads')) {
+      totalUploads = storage.totalUploads;
+    }
```

Comment:
we've got two blocks with this pattern. this is initialization code so i suggest moving them into the `Storage` constructor. Also check that they're necessary at all because I think the getters (as implemented) will return 0 when it's not already set.